### DISCUSSION TITLE
List of OAS v3 issues addressed by this proposal

### DISCUSSION THREAD

**Discussion by darrelmiller** [OAI/sig-moonwalk#21] [General]
- https://github.com/OAI/OpenAPI-Specification/issues/182 
This is now supported by named request objects. 

- https://github.com/OAI/OpenAPI-Specification/issues/1459
This is supported by the use of UriTemplates

- https://github.com/OAI/OpenAPI-Specification/issues/445
- https://github.com/OAI/OpenAPI-Specification/issues/256
These are supported by using a JSONSchema to define parameters

- https://github.com/OAI/OpenAPI-Specification/issues/2316
This is supported by dropping the `path` leading slash requirement

**Comment by nebulon42** [comment]
Suggest to add https://github.com/OAI/OpenAPI-Specification/issues/521.

**Comment by handrews** [comment]
I've added a ["Moved to Moonwalk"](https://github.com/OAI/OpenAPI-Specification/issues?q=label%3A%22Moved+to+Moonwalk%22) label for issues that are only going to be addressed here (if at all).  I am including issues that are clearly related to articulated Moonwalk proposals or principles, even if it is not clear whether that specific request will end up being handled. 

This seems like it may be a little more manageable than keeping this list updated.
